specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/stable/2016-05-01/Alert.json

```diff
@@ -56,24 +62,27 @@
                 ]
             }
         },
-        "/subscriptions/{subscriptionId}/resourceGroups/System.{location}/providers/Microsoft.InfrastructureInsights.Admin/regionHealths/{location}/alerts/{alertName}": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.InfrastructureInsights.Admin/regionHealths/{location}/alerts/{alertName}": {
             "get": {
                 "x-ms-examples": {
-                    "Get an alert": {
-                        "$ref": "./examples/Alert/AlertGetOperation_example.json"
+                    "Returns the requested an alert.": {
+                        "$ref": "./examples/Alert/Get.json"
                     }
                 },
                 "tags": [
                     "Alerts"
                 ],
-                "description": "Get an alert.",
+                "description": "Returns the requested an alert.",
                 "operationId": "Alerts_Get",
                 "parameters": [
                     {
                         "$ref": "InfrastructureInsights.json#/parameters/SubscriptionIdParameter"
                     },
                     {
-                        "$ref": "InfrastructureInsights.json#/parameters/LocationParameter"
+                        "$ref": "InfrastructureInsights.json#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "InfrastructureInsights.json#/parameters/RegionParameter"
```

Comment:
>RegionParameter [](start = 73, length = 15)

Is this paramater required? it does not look like it

Reply:
yes it is required, it is the {location} parameter in the URI.

Reply:
okok, nevermind

---
In reply to: [181146243](https://github.com/Azure/azure-rest-api-specs/pull/2812#discussion_r181146243) [](ancestors = 181146243)